sound/hda/intel-nhlt.c

```diff
@@ -55,16 +55,23 @@ int intel_nhlt_get_dmic_geo(struct device *dev, struct nhlt_acpi_table *nhlt)
 
 		/* find max number of channels based on format_configuration */
 		if (fmt_configs->fmt_count) {
+			struct nhlt_fmt_cfg *fmt_cfg;
```

Comment:
@ujfalusi where are you handling this "At the same time prepare the code for converting the fmt_config to a u8
flexible array to prevent further abuse."?

Reply:
@ujfalusi is this PR still current? No update and reply to @ranj063 's question.